examples/plotting/file/periodic_shells.py

```diff
@@ -69,11 +80,10 @@ def bohr_diagram():
     plot = figure(
         width=150, height=150,
         x_axis_type=None, y_axis_type=None,
+        x_range = (-8,8), y_range = (-8,8),
         toolbar_location=None, outline_line_color=None,
         match_aspect=True,
     )
-    plot.x_range.only_visible = True
-    plot.y_range.only_visible = True
```

Comment:
@mosc9575 what was the motivation for this change? AFAIK `only_visible` is not demonstrated anywhere else and it might be nice to have at least on example to point to

Reply:
These two lines are throwing an error if I add `x_range` and `y_range`. 

> AttributeError: unexpected attribute 'only_visible' to Range1d

Here is the old version:

![old](https://user-images.githubusercontent.com/68053396/164623866-dcc94175-fa32-4073-b8e1-609104ff268f.png)

And here is how it looks with the ranges.

![new](https://user-images.githubusercontent.com/68053396/164623858-2d1dd782-1e92-4e31-a867-93f4adbbc080.png)